frontend/package.json

```diff
@@ -36,7 +36,7 @@
     "react-paginate": "6.3.2",
     "react-redux": "7.2.9",
     "react-redux-loading-bar": "4.6.0",
-    "react-resize-detector": "5.2.0",
+    "react-resize-detector": "7.1.2",
```

Comment:
The leaflet map component would not consistently zoom and center to the selected property.